common/utils/analytics.js

```diff
@@ -0,0 +1,49 @@
+import ReactGA from 'react-ga';
+
+type AnalyticsCategory = 'collections' | 'editorial' | 'public-programme';
+type Props = {|
+  category: AnalyticsCategory,
+  seriesUrl: ?string,
+  positionInSeries: ?string,
```

Comment:
Could we remove these until we get to the editorial app where we can actually question if we need them?